src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -198,12 +209,21 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
         /// </summary>
         public void Start()
         {
-            if (preferences!= null && preferences.IsAnalyticsReportingApproved)
+            if (preferences != null && 
+                (preferences.IsAnalyticsReportingApproved || preferences.IsADPAnalyticsReportingApproved))
             {
                 //If not ReportingAnalytics, then set the idle time as infinite so idle state is not recorded.
-                Service.StartUp(product, new UserInfo(Session.UserId), preferences.IsAnalyticsReportingApproved ? TimeSpan.FromMinutes(30) : TimeSpan.MaxValue);
```

Comment:
I removed the backup value `TimeSpan.MaxValue` since it seems it was never hit before...

Reply:
@pinzart Yes, now I remember because we made the change that if GA is not agreed, we do not initialize the client at all. It seems this change will undo that previous change though. e.g. if the ADP is agreed on, the service will be started regardless

Reply:
@QilongTang 
`
 ADP is agreed on, the service will be started regardless
`
That is correct. My intention was to make ADP and Google independent of each other. So if a user opts into ADP (but not Google) ...then Analytics service will be started (but only ADPTracker will process events). Similar if Google is opted in but ADP is not.